src/main/java/com/spikes2212/dashboard/AutoChooser.java

```diff
@@ -0,0 +1,144 @@
+     * @param defaultOption the default command this {@link SendableChooser} will use as the default option
+     * @param options the commands that will be added to this {@link SendableChooser} other than the default command
+     */
+    public AutoChooser(String rootNamespaceName, Command defaultOption, Command... options) {
+        rootNamespace = new RootNamespace(rootNamespaceName);
+        String defaultName = defaultOption.getClass().getSimpleName();
+        addName(defaultName);
+        setDefaultOption(defaultName, defaultOption);
+        for (Command option : options) {
+            addOption(option.getClass().getSimpleName(), option);
+        }
+        putOnShuffleboard();
+    }
+
+    /**
+     * Creates an autonomous chooser from the given commands, where the first command will be the default option and the
+     * rest of the commands will be the rest of the options.
+     *
+     * <p>All options will have names matching their command's {@link Class#getSimpleName()}. In case of multiple
+     * instances of the same class, numbers will also be added to differentiate between the names.</p>
+     *
+     * @param defaultOption the default command this {@link SendableChooser} will use as the default option
+     * @param options the commands that will be added to this {@link SendableChooser} other than the default command
+     */
+    public AutoChooser(Command defaultOption, Command... options) {
+        this(DEFAULT_NAMESPACE_NAME, defaultOption, options);
+    }
+
+    /**
+     * Creates an autonomous chooser where the first {@link Command} and {@link String} will be used as the default
+     * option, and the rest of the options will be parsed as following: every even index (0,2,4,...) will be a command
+     * and every odd index (1,3,5,...) will be used as the name for the previous command.
+     *
+     * @param rootNamespaceName the name that will be given to the {@link RootNamespace}
+     * @param defaultOption the default command this {@link SendableChooser} will use as the default option
+     * @param defaultOptionName the name for the default command
+     * @param options the rest of the options, where every even index is a command to be added and every odd index is
+     *               the name for the previously mentioned command
+     */
+    public AutoChooser(String rootNamespaceName, Command defaultOption, String defaultOptionName, Object... options) {
+        rootNamespace = new RootNamespace(rootNamespaceName);
+        setDefaultOption(defaultOptionName, defaultOption);
+        Command command;
+        String name;
+        for (int i = 0; i < options.length - options.length % 2; i += 2) {
+            if (options[i] instanceof Command) {
+                command = (Command) options[i];
+            } else {
+                throw new IllegalArgumentException("The " + (i + 3) + " argument is not a Command.");
```

Comment:
add "th" after the number to indicate ordinality

Reply:
if you deem it necessary, use an algorithm to determine when to use 'st', 'nd' or 'rd' instead of 'th'